### PR TITLE
docs: add package manager config page & update runtime

### PR DESCRIPTION
**Pull request by TomChv**
Add `packageManager` config doc page.
Update runtime page with version configuration.

### CODE REVIEW

File: docs/current_docs/manuals/developer/package-managers.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+If no version is specified, `yarn@1.22.22` is used by default.
+
+If you use Yarn 3.0 or above, the TypeScript SDK will also generate a `.yarnrc.yml` file in your module's root directory. This file is used to configure the Yarn package manager linker to `node_modules`.
+
+This is required to correctly resolve `@dagger.io/dagger` as local dependencie of your module.
+
+```yaml
+nodeLinker: node-modules
+```
```
Comment (gerhard) [inline docs/current_docs/manuals/developer/package-managers.mdx:44]: Where is this property set?

File: docs/current_docs/manuals/developer/package-managers.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+If no version is specified, `pnpm@8.15.4` is used.
+
+The TypeScript SDK will also generate a `pnpm-workspace.yml` file in your module's root directory. This file is used to configure the Pnpm package manager.
+
+This is required to correctly resolve `@dagger.io/dagger` as local dependencie of your module.
+
+```yaml
+packages:
+  - './sdk'
+```
```
Comment (gerhard) [inline docs/current_docs/manuals/developer/package-managers.mdx:63]: Where is this property set?

**Review by gerhard — APPROVED** [review]
This looks good to me now @TomChv 👍

I've mentioned two nits, but otherwise OK to merge. No need to block the PR on those if we want to keep moving with this.

File: docs/current_docs/manuals/developer/package-managers.mdx (hunk 1 above)
Comment (TomChv) [reply docs/current_docs/manuals/developer/package-managers.mdx:44]: Explained on the doc page above

**Comment by vikram-dagger** [comment]
@TomChv @gerhard does this new content overlap with https://docs.dagger.io/manuals/developer/module-structure#language-native-packaging and/or https://docs.dagger.io/manuals/developer/language-dependencies ?

File: docs/current_docs/manuals/developer/package-managers.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+slug: /manuals/developer/typescript/784253/package-manager
+displayed_sidebar: "current"
+toc_max_heading_level: 2
+title: "Alternative Package Managers"
+---
+
+# Alternative Package Managers
+
+:::important
+This page is only applicable to the TypeScript SDK. Alternative package managers are not currently supported in the Python and Go SDKs.
```
Comment (vikram-dagger) [inline docs/current_docs/manuals/developer/package-managers.mdx:11]: AFAIK you can still use `poetry` and `pip` in Python, so does that impact this note @TomChv ?

**Pull request merged**